src/content/self-hosted/install/releases.mdx

```diff
@@ -5,7 +5,46 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+
+## 1.4.0
+<TODO: insert release date here once it's published>
+
+### Features
+- Update the Destroy All UI 
+- Support custom resources (requests and limits) for installer jobs
+- Render External Resource notes using Markdown
```

Comment:
I would document a single feature, "external resources", explaining what they are and how to use it.
@jLopezbarb @viddo do we have the field `external` documented for the okteto manifest?

Reply:
The url reference should work once we merge @jLopezbarb 's pr